workspace/suid-interpreter.nix

```diff
@@ -0,0 +1,149 @@
+{ pkgs }:
+
+let
+  suidSource = pkgs.writeText "suid_interpreter.c" ''
```

Comment:
Isn't this long enough (and security critical enough) to be it's own file, with syntax highlighting and all? I know it's kinda hard to fit into the file tree logically

Reply:
Yeah you convinced me, a good location was the thing stopping me. For now I've added a `core` directory to make the structure a little better.